go/client/cmd_config.go

```diff
@@ -103,11 +103,25 @@ func (v *CmdConfigSet) ParseArgv(ctx *cli.Context) error {
 			return fmt.Errorf("Missing string value argument")
 		}
 		s := args[1]
+		if !ctx.IsSet("string") && v.looksLikeBool(s) {
```

Comment:
You said you wanted `keybase config set -s foo t` to work and not `keybase config set foo t`.